application/app/services/download/command/response.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module Download::Command
+  class Response
+    attr_reader :status, :headers, :body
+
+    def initialize(status: 200, headers: {}, body: {})
+      @status = status
+      @headers = headers || {}
+      @body = OpenStruct.new(body)
+    end
+
+    def to_json(*_args)
+      {
+        status: status,
+        headers: headers,
+        body: body.to_h
+      }.to_json
+    end
```

Comment:
why do you need *_args in this method?

Reply:
I think that the Rails method signature for to_json takes an argument. I think rails call it `options`. I just wanted to follow that. But will remove it to make it consistent with our other `to_json` methods

Reply:
fixed